File: 백준/Silver/2751. 수 정렬하기 2/수 정렬하기 2.java
import java.io.BufferedReader;
import java.io.BufferedWriter;
import java.io.IOException;
import java.io.InputStreamReader;
import java.io.OutputStreamWriter;
import java.util.Arrays;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
        int tc = Integer.parseInt(br.readLine());

        int[] array = new int[tc];

        for(int i = 0; i < tc; i++){
            array[i] = Integer.parseInt(br.readLine());
        }

        br.close();

        Arrays.sort(array);
        StringBuilder sb = new StringBuilder();

        for(int number: array){
            sb.append(number).append("\n");
        }
        bw.write(String.valueOf(sb));
        bw.flush();
        bw.close();

    }
}

File: 백준/Silver/2751. 수 정렬하기 2/수 정렬하기 2.kt
import java.io.*

fun main() = with(BufferedReader(InputStreamReader(System.`in`))) {

    val N = readLine().toInt()
    val set = sortedSetOf<Int>()

    repeat(N) {
        set.add(readLine().toInt())
    }
    close()

    val bw = BufferedWriter(OutputStreamWriter(System.out))
    set.forEach {
        bw.write("$it\n")
    }
    bw.flush()
    bw.close()
}